concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2590,6 +2591,8 @@ jobs:
           file: paas-cf/concourse/tasks/generate-test-config.yml
           params:
             TEST_PROPERTIES: smoke_tests
+            APP_DOMAIN: ((apps_dns_zone_name))
```

Comment:
the APP_DOMAIN and SYSTEM_DOMAIN params have to be added to the 3 other generate-test-config calls.
Fix commit: bb33f244f87c4e89d1f3329ca7e6e720bef7963f